src/applications/mhv/secure-messaging/components/shared/AlertBackgroundBox.jsx

```diff
@@ -66,7 +124,21 @@ const AlertBackgroundBox = props => {
           }}
         >
           <div>
-            <p className="vads-u-margin-y--0">{activeAlert.content}</p>
+            {alertContent === SERVER_ERROR_503 ? (
+              <p className="vads-u-margin-y--0" data-testid="alert-text">
+                {alertContent}
+                <h1 className="sr-only" aria-live="polite" aria-atomic="true">
+                  {alertAriaLabel}
+                </h1>
+              </p>
+            ) : (
+              <p className="vads-u-margin-y--0" data-testid="alert-text">
+                {alertContent}
+                <span className="sr-only" aria-live="polite" aria-atomic="true">
+                  {alertAriaLabel}
+                </span>
+              </p>
+            )}

```

Comment:
these lines look very similar, we may want to consider refactoring of this statement 
```suggestion

```

Reply:
good call out -- I can make this smaller.